tests/parser/markup/missing_closing_brace_after_self_close.mf2

```diff
@@ -0,0 +1,26 @@
+{#foo /
+=== spans ===
+                    {#foo /
```

Comment:
Can you add a test like `{#foo /hello` and `{#foo / hello`? Let's always add a panicking test when we introduce a new panic/todo.